Jenkinsfile

```diff
@@ -47,7 +47,7 @@ pipeline {
           rtMaven.deployer.deployArtifacts = false
 
           withCredentials([string(credentialsId: 'gpg-signing-key-passphrase', variable: 'PASSPHRASE')]) {
-            buildInfo = rtMaven.run pom: 'pom.xml', goals: 'clean compile javadoc:jar source:jar install -P sign,git,withDep -Dmaven.repo.local=.m2 $MAVEN_TEST_OPTIONS -Dgpg.passphrase=$PASSPHRASE'
+            buildInfo = rtMaven.run pom: 'pom.xml', goals: 'clean compile javadoc:jar source:jar install -P sign,git,withDep -Dmaven.repo.local=.m2 $MAVEN_TEST_OPTIONS -Dgpg.passphrase=$PASSPHRASE -DdetectJavaApiLink=false'
```

Comment:
` -DdetectJavaApiLink=false` should be added to all `rtMaven.run` where `javadoc:…` is used. Maybe with a preceding comment that this should be removed in the future.

Reply:
this was just a temporary "hotfix" to make it compile, so I can see jenkins' javadoc error messages. since the branch wouldn't trigger the deploy/publish-javadoc stages, it is enough to just do the hotfix here. Anyway, this wasn't supposed to get merged into master from this branch.